pna.p4

```diff
@@ -743,6 +743,35 @@ extern void update_expire_info(
     in ExpireTimeProfileId_t expire_time_profile_id);
 
 
+// Set the expire time of the matched entry in the table to the value
+// specified in the parameter expire_time_profile_id, if the value
+// parameter is equal to the data parameter. If the value and data
+// parameters are not equal, the function has no effect.
+//
+// @param D          Tuple of fields of data to match with value.
+// @param data       Data that is matched to the value parameter
+//                   to determine if the expire time will be set.
+// @param value      Value that is matched with corresponding
+//                   data parameter.
+// @expire_time_profile_id
+//                   The expire time to set for the matched entry,
+//                   if the data and value parameters are equal.
+//
+// Examples:
+// set_expire_time_if({hdr.tcp.flags, meta.direction},
+//                    {TCP_FLG_SYN, OUTBOUND},
+//                    tcp_connection_start_time_profile_id);
```

Comment:
If the new parameters for `set_expire_time_if` were a single additional parameter `in bool condition`, then the example above could be written as shown below:
```
set_expire_time_if((hdr.tcp.flags == TCP_FLG_SYN) && (meta.direction == OUTBOUND),
                              tcp_connection_start_time_profile_id);
```